### ISSUE TITLE
Vim Cheat Sheet: 

### ISSUE THREAD

**Issue by tudorsaitoc** [duckduckgo/zeroclickinfo-goodies#3321]
Heading format seems off (details below):

Was the intention for the title to be displayed on one line?

I'm getting two lines like this:
```
[Vim Cheat Shee](https://duck.co/ia/view/vim_cheat_sheet)
[t](https://duck.co/ia/view/vim_cheat_sheet)
```

------
IA Page:  http://duck.co/ia/view/vim_cheat_sheet 
[Maintainer](http://docs.duckduckhack.com/maintaining/guidelines.html): @NaveenKarippai

**Comment by NaveenKarippai** [comment]
@tudorsaitoc could you please be more specific. Are you referring to subtitle here? If possible, can we have a screenshot of the issue. Thanks!

**Comment by zekiel** [comment]
@tudorsaitoc  I don't see the same thing but maybe it's browser-specific. Can you confirm where the title is going to two lines? 

Another note is that the subtitle is a bit odd as "VI Improved." Maybe that can be updated if someone's tinkering around in there. 
![vim cheat sheet subtitle](https://cloud.githubusercontent.com/assets/1846953/16492707/b022920a-3eb0-11e6-88d3-26b0110a0d95.png)

**Comment by NaveenKarippai** [comment]
@zekiel agreed! The (above) description was taken from vim manpage :small_blue_diamond:

**Comment by tudorsaitoc** [comment]
I'm on Chrome.  This is how it shows up:

[screenshots 2016-06-30 at 3.15.30 PM.pdf](https://github.com/duckduckgo/zeroclickinfo-goodies/files/342172/screenshots.2016-06-30.at.3.15.30.PM.pdf)

**Comment by zekiel** [comment]
@tudorsaitoc  ahhh, ok. Thanks for the screenshot! That's the informational page for the Instant Answer which shows the metadata, description, and example queries of that Instant Answer. The actual answer on DuckDuckGo is what we thought you meant! 

IA in Action: https://duckduckgo.com/?q=vim+cheat+sheet&ia=cheatsheet&iax=1
IA Page: https://duck.co/ia/view/vim_cheat_sheet

What you pointed out is definitely an issue, though! I can see it as well via Chrome. Updating the title here and assigning to someone for investigation:
![heading](https://cloud.githubusercontent.com/assets/1846953/16522565/a34aeb10-3f6c-11e6-8a4c-7f734f76ea17.png)

**Comment by akanshgulati** [comment]
@moollaza I believe some issue came while creating the IA. Any update @jagtalon ?

**Comment by rwsender** [comment]
@zekiel I have fixed the issue with the subtitle, it seems the title issue is no longer occurring but?
![image](https://cloud.githubusercontent.com/assets/9061301/20568411/630ec798-b194-11e6-999d-82127f4c69ea.png)
